### PR TITLE
Release docs dates

### PR DESCRIPTION
**Pull request by guidari**
Update release docs to match the new schedule on [release radar](https://github.com/carbon-design-system/carbon/wiki/Release-radar)

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ checkpoints:
 
 ### Prerelease
 
-The prerelease occurs on the first Tuesday of a sprint. During this stage, the
+The prerelease occurs on the last Monday of a sprint. During this stage, the
```
Comment (alisonjoseph) [inline docs/release.md:71]: I think its the first Monday, since sprints start on a Thursday?

File: docs/release.md [hunk 2]
```diff
@@ -121,10 +121,9 @@ To do so, follow the above steps for [Prerelease](#prerelease) but specify
 
 ### Stable release
 
-A stable release occurs on the first Thursday of a sprint and finishes on the
-morning of the first Friday of a sprint. This should occur after the prerelease
-has been tested and validated. During this stage, the release team will do the
-following:
+A stable release occurs on the last Wednesday and finishes later in the day.
```
Comment (alisonjoseph) [inline docs/release.md:124]: Same thing here, I think its first Wednesday

File: docs/release.md (hunk 1 above)
Comment (tay1orjones) [reply docs/release.md:71]: Current releases straddle the sprint end/beginning. Prerelease on tuesday, sprint ends wednesday, stable release on Thursday. I just moved the dates in the release radar earlier by 1 day, so it's the last Monday/Wednesday of the sprint.

**Review by tay1orjones — APPROVED** [review]

File: docs/release.md (hunk 1 above)
Comment (alisonjoseph) [reply docs/release.md:71]: Oops, I had the sprint dates wrong on my calendar 🤦‍♀️

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**